pkg/operator/ceph/cluster/osd/config.go

```diff
@@ -91,6 +106,49 @@ func (c *Cluster) osdRunFlagTuningOnPVC(osdID int) error {
 	return nil
 }
 
+func (c *Cluster) osdRunFlagFastTuningOnPVC(osdID int) error {
+	who := fmt.Sprintf("osd.%d", osdID)
+	do := make(map[string]string)
+
+	// Default value of osd_op_num_threads_per_shard for SSDs
+	do["osd_op_num_threads_per_shard_ssd"] = osdOpNumThreadsPerShardSSD
```

Comment:
lines 114-138 are not needed if using a map as suggested in the previous comment

Reply:
Done.